tests/Integration/ManagementTest.php

```diff
@@ -0,0 +1,197 @@
+        $this->assertNotEmpty($merchant['id']);
+        $this->assertEquals("Active", $merchant['status']);
+        $this->assertEquals($merchantId, $merchant['id']);
+    }
+
+    /**
+     * Get /me
+     *
+     * @throws \Adyen\AdyenException
+     */
+    public function testGetMe()
+    {
+        $response = $this->management->me->retrieve();
+        $this->assertNotEmpty($response);
+        $this->assertNotEmpty($response['id']);
+        $this->assertNotEmpty($response['username']);
+        $this->assertNotEmpty($response['clientKey']);
+        $this->assertNotEmpty($response['roles']);
+        $this->assertTrue($response['active']);
+        $this->assertNotEmpty($response['companyName']);
+    }
+
+    /**
+     * Get /companies
+     *
+     * @throws \Adyen\AdyenException
+     */
+    public function testGetCompanies()
+    {
+        $response = $this->management->companyAccount->list();
+        $this->assertNotEmpty($response);
+        $this->assertNotEmpty($response[self::LINKS]);
+        $this->assertNotEmpty($response['data']);
+        $this->assertNotEmpty($response[self::ITEMS_TOTAL]);
+        $this->assertTrue($this->count($response['data']) > 0);
+    }
+
+    /**
+     * Get /companies/{id}
+     *
+     * @throws \Adyen\AdyenException
+     */
+    public function testGetCompanyById()
+    {
+        $companies = $this->management->companyAccount->list();
+        $companyId = $companies["data"][0]["id"];
+        $response = $this->management->companyAccount->retrieve($companyId);
+        $this->assertNotEmpty($response);
+        $this->assertNotEmpty($response['id']);
+        $this->assertNotEmpty($response['name']);
+        $this->assertNotEmpty($response['status']);
+    }
+
+    /**
+     * Get /companies/{companyId}/webhooks
+     *
+     * @throws \Adyen\AdyenException
+     */
+    public function testGetCompanyWebhooksById()
+    {
+        $companies = $this->management->companyAccount->list();
+        $companyId = $companies["data"][0]["id"];
+        $response = $this->management->companyWebhooks->retrieve($companyId);
+        $this->assertNotEmpty($response);
+        $this->assertNotEmpty($response[self::LINKS]);
+        $this->assertNotEmpty($response['data']);
+        $this->assertNotEmpty($response[self::ITEMS_TOTAL]);
+        $this->assertNotEmpty($response['pagesTotal']);
+        $this->assertTrue($this->count($response['data']) > 0);
+    }
+
+    /**
+     * Post /merchants/{merchantId}/webhooks/
+     *
+     * @throws \Adyen\AdyenException
+     */
+    public function testCreateMerchantWebhooks()
+    {
+        if (empty($this->settings['merchantAccount']) ||
+            $this->settings['merchantAccount'] == self::YOUR_MERCHANT_ACCOUNT) {
+            $this->skipTest(self::SKIP_TEST_MESSAGE);
+            return null;
+        }
+        $params = array(
+            "type" => "standard",
+            "url" => "https://magento2-devx.ataberkylmz.com/",
```

Comment:
Can we supply a non working URL here instead or does this service also test the webhook being accepted?